src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/general/GeneralCommands.java

```diff
@@ -65,6 +62,17 @@ public void main(CommandSender sender) {
         }
     }
 
+    @Subcommand("%locale")
+    @CommandPermission("simpleclans.anyone.locale")
+    @Description("{@@command.description.locale}")
+    @CommandCompletion("@locales")
+    public void locale(ClanPlayer cp, @Name("locale") String locale) {
```

Comment:
`@Values("@locales")` to validate the input based on the completion list

Reply:
Nice 🤔 👍🏻

Reply:
No, you